pkg/genutil/ensure.go

```diff
@@ -13,16 +17,18 @@ func MustEnsureUInt32(value int) uint32 {
 
 // EnsureUInt32 ensures that the specified value can be represented as a uint32.
 func EnsureUInt32(value int) (uint32, error) {
```

Comment:
These functions are basically doing `safecast` things anyway, so I rewrote them in terms of safecast to silence the errors. We can upstream it later.